lib/connection_pool/timed_stack.rb

```diff
@@ -169,9 +170,9 @@ def fetch_connection(options = nil)
   def shutdown_connections(options = nil)
     while connection_stored?(options)
       conn = fetch_connection(options)
+      @created -= 1 unless @created == 0
       @shutdown_block.call(conn)
     end
-    @created = 0
```

Comment:
This changes the return value. Is this a concern?

Reply:
For the public APIs, we should probably define useful return values or return 'nil'.

Reply:
>This changes the return value. Is this a concern?

This is private method, and we currently don't do anything with the return value. So there shouldn't be any problem with this change.

>For the public APIs, we should probably define useful return values or return 'nil'.

`TimedStack#reap` and `TimedStack#shutdown` both currently return nil unless an error is raised 👍